.github/scripts/run-librispeech-lstm-transducer-stateless2-2022-09-03.yml

```
@@ -204,6 +204,46 @@ if [[ x"${GITHUB_EVENT_LABEL_NAME}" == x"shallow-fusion" ]]; then
     --rnn-lm-tie-weights 1
 fi
 
+if [[ x"${GITHUB_EVENT_LABEL_NAME}" == x"LODR" ]]; then
+  bigram_repo_url=https://huggingface.co/marcoyang/librispeech_bigram
+  log "Download bi-gram LM from ${bigram_repo_url}"
+  git clone $bigram_repo_url
+  bigramlm_repo=$(basename $bigram_repo_url)
+  pushd bigramlm_repo
+  git lfs pull --include "2gram.fst.txt"
+  ln -s 2gram.fst.txt $repo/data/lang_bpe_500/.
+  popd
+
+  lm_repo_url=https://huggingface.co/ezerhouni/icefall-librispeech-rnn-lm
+  log "Download pre-trained RNN-LM model from ${lm_repo_url}"
+  git clone $lm_repo_url
```

```suggestion
  GIT_LFS_SKIP_SMUDGE=1 git clone $lm_repo_url
```

Please add `GIT_LFS_SKIP_SMUDGE=1` in the new code.
I have not found time to update the old code.